daemons/controld/controld_schedulerd.c

```diff
@@ -469,10 +478,8 @@ do_pe_invoke_callback(xmlNode * msg, int call_id, int rc, xmlNode * output, void
     } else if (num_cib_op_callbacks() > 1) {
         crm_debug("Re-asking for the CIB: %d other peer updates still pending",
                   (num_cib_op_callbacks() - 1));
-        sleep(1);
-        controld_set_fsa_action_flags(A_PE_INVOKE);
-        controld_trigger_fsa();
-        return;
+        timer = mainloop_timer_add("timer", 1000, FALSE, sleep_timer, NULL);
```

Comment:
The first argument ("timer") here should probably be something a little more descriptive - "cib_retry" for instance.